README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+capstone1-concert-website
 
 > Description the project.
```

Comment:
- [ ] Kindly ensure that you write a proper description for your capstone project so as to make your project can be professional. Your readme will give recruiters insight as to what your project is all about so it okay to market your project well 😄.